wdl/src/main/scala/wdl/types/WdlFlavoredWomType.scala

```diff
@@ -0,0 +1,57 @@
+package wdl.types
+
+import scala.collection.JavaConverters._
+import wdl.{WdlExpression, WdlSyntaxErrorFormatter}
+import wdl.AstTools.EnhancedAstNode
+import wom.core.WorkflowSource
+import wom.types.{WdlBooleanType, WdlFloatType, WdlIntegerType, WdlType}
+import wom.types.WdlType.parser
+import wom.values.{WdlBoolean, WdlFloat, WdlInteger, WdlValue}
+
+object WdlFlavoredWomType {
```

Comment:
This grossness was moved out of `WdlType` and is used only for migrations and tests.

Reply:
Can in be in `wdl/src/test`?

Reply:
No the production code migrations use both methods here unfortunately.

Reply:
Oh, I missed the `"migrations and"` part of your comment. Oops!